tools/docker/Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM ubuntu:16.04
+
+# No interactive frontend during docker build
+ENV DEBIAN_FRONTEND=noninteractive \
+    DEBCONF_NONINTERACTIVE_SEEN=true
+
+# General requirements not in the base image
+RUN apt-get -qqy update \
+  && apt-get -qqy install \
+    bzip2 \
+    ca-certificates \
+    dbus-x11 \
+    git \
+    locales \
+    pulseaudio \
+    python \
+    python-pip \
+    tzdata \
+    sudo \
+    unzip \
+    wget \
+    xvfb
+
+# Installing just the deps of firefox and chrome is moderately tricky, so
+# just install the default versions of them, and some extra deps we happen
+# to know that chrome requires
+
+RUN apt-get -qqy install \
+    firefox \
+    chromium-browser \
```

Comment:
IIUC, the extra dependencies below are dependencies of Chrome (that you happen to know) but not of chromium-browser.

IMHO, since Chrome is already distributed in a deb package, we can perhaps skip this (i.e. only install firefox). See my next comment in `start.sh` for dependency problems.